addons/im_livechat/controllers/main.py

```diff
@@ -186,17 +187,20 @@ def get_session(self, channel_id, anonymous_name, previous_operator_id=None, cha
             channel = channel.with_context(guest=guest)  # a new guest was possibly created
             if not chatbot_script or chatbot_script.operator_partner_id != channel.livechat_operator_id:
                 channel._broadcast([channel.livechat_operator_id.id])
+            if guest:
+                store.add_global_values(guest_token=guest._format_auth_cookie())
+        request.env["res.users"].with_context(guest=guest)._init_store_data(store)
+        guest._bus_send_store(store)
+        if channel:
+            # Make sure not to send the channel on the guest bus, otherwise
+            # "isLoaded" value could be overwritten.
             store.add(
                 channel,
```

Comment:
Wouldn't we still want to at least send the channel itself for cross-tab?

Reply:
We could, but it might still result in a fetch if chat window opens before the reception of the bus payload.